app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessToilets.kt

```diff
@@ -28,6 +29,12 @@ class AddWheelchairAccessToilets : OsmFilterQuestType<WheelchairAccessToiletsAns
     override fun createForm() = AddWheelchairAccessToiletsForm()
 
     override fun applyAnswerTo(answer: WheelchairAccessToiletsAnswer, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
-        answer.applyTo(tags)
+        when (answer) {
+            is WheelchairAccessToilets -> {
+                tags.updateWithCheckDate("wheelchair", answer.access.osmValue)
+                tags["toilets"] = "yes"
```

Comment:
Is adding `toilets = yes` on `amenity = toilets` deliberate?

Reply:
Ah, no, this hasn't gone so well. We shouldn't offer a no toilet option when asking about `amenity=toilets` anyway. I'll fix this